src/PicOfTheDay.vue

```diff
@@ -1,36 +1,37 @@
 <template> 
   <section class='potd-component'>
     <iframe
-      :src="url" 
+      :src="potd.url" 
       class='video' 
-      v-if="media_type === 'video'" 
-      :title="title"
+      v-if="potd.media_type === 'video'" 
+      :title="potd.title"
       allowfullscreen="allowfullscreen"
       mozallowfullscreen="mozallowfullscreen" 
       msallowfullscreen="msallowfullscreen" 
       oallowfullscreen="oallowfullscreen" 
       webkitallowfullscreen="webkitallowfullscreen"
     />
-    <img :src="url" class='photo' v-if="media_type === 'image'" :alt="title">
-    <h2>{{ date }}</h2>
-    <h3>{{ title }}</h3>
-    <h4>{{ explanation }}</h4>
+    <img :src="potd.url" class='photo' v-if="potd.media_type === 'image'" :alt="potd.title">
+    <h2>{{ potd.date }}</h2>
+    <h3>{{ potd.title }}</h3>
+    <h4>{{ potd.explanation }}</h4>
   </section>
 </template>
 
 <script>
 export default {
-  name: "PicOfTheDay",
-  data() {
-    return {
-      date: "2021-02-23",
-      explanation: "What would it look like to land on Mars?  To better monitor the instruments involved in the Entry, Decent, and Landing of the Perseverance Rover on Mars last week, cameras with video capability were included that have now returned their images. The featured 3.5-minute composite video begins with the opening of a huge parachute that dramatically slows the speeding spacecraft as it enters the Martian atmosphere.  Next the heat shield is seen separating and falls ahead. As Perseverance descends, Mars looms large and its surface becomes increasingly detailed.  At just past 2-minutes into the video, the parachute is released and Perseverance begins to land with dust-scattering rockets. Soon the Sky Crane takes over and puts Perseverance down softly, then quickly jetting away. The robotic Perseverance rover will now begin exploring ancient Jezero Crater, including a search for signs that life once existed on Earth's neighboring planet.",
-      media_type: "video",
-      service_version: "v1",
-      title: "Video: Perseverance Landing on Mars",
-      url: "https://www.youtube.com/embed/4czjS9h4Fpg?rel=0"
-    }
-  }
+  name: 'PicOfTheDay',
+  props: ['potd'],
```

Comment:
Are props always in an array?

Reply:
Yep! And they should always be put between quotes